staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -256,5 +273,11 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"during this window all incoming requests will be rejected with a status code 429 and a 'Retry-After' response header, "+
 		"in addition 'Connection: close' response header is set in order to tear down the TCP connection when idle.")
 
+	// NOTE: this option is applicable to Microshift only, this should never be enabled for OCP.
+	fs.StringVar(&s.NotReadyDebuggerGroup, "not-ready-debugger-group", s.NotReadyDebuggerGroup, ""+
```

Comment:
Do we really want that to be configurable? Having administrators being able to change the name of the debugging group will just make it harder for tools to assume which user group they should assume IMHO.

Reply:
yeah, I think we discussed this, I have hardcoded the group name to `system:openshift:risky-not-ready-microshift-debugging-group`